lib/bridge/IrcHandler.js

```diff
@@ -1112,6 +1130,51 @@ IrcHandler.prototype._invalidateNickUserIdMap = function(server, channel) {
     this.nickUserIdMapCache.delete(`${server.domain}:${channel}`);
 }
 
+IrcHandler.prototype._handleLeaveQueue = async function(item) {
+    const bridge = this.ircBridge.getAppServiceBridge();
+    let retryRooms = [];
+    for (const room of item.rooms) {
+        const roomId = room.getId();
+        item.req.log.info(
+            "Leaving room %s (%s)",
+            roomId,
+            item.userId
```

Comment:
trailing comma (consistency)
```suggestion
            item.userId,
```